apps/web-giddh/src/app/invoice/settings/invoice.settings.component.html

```diff
@@ -677,7 +691,7 @@ <h4 class="clearfix bd-rb font-16">
                                                 </mat-slide-toggle>
                                                 <label class="left-gap">{{ localeData?.auto_mail_update }}</label>
                                             </div>
-                                            <div class="col-sm-12 col-md-9 form-group">
+                                            <div class="col-sm-12 col-md-9 form-group d-flex column-gap-1">
```

Comment:
```suggestion
                                            <div class="col-sm-12 col-md-9 form-group d-flex column-gap1">
```

not class present in giddh 'column-gap-1'